perf/perf-producer.go

```diff
@@ -98,7 +98,7 @@ func produce(produceArgs *ProduceArgs, stop <-chan struct{}) {
 
 	ch := make(chan float64)
 
-	go func() {
+	go func(stop <-chan struct{}) {
```

Comment:
maybe rename this the stopCh so it's clear in the go routine which one it's using? Just curious what are the steps for it getting stuck?

Reply:
> maybe rename this the stopCh so it's clear in the go routine which one it's using?

ok

> Just curious what are the steps for it getting stuck?

```
func produce(produceArgs *ProduceArgs, stop <-chan struct{}) {
    .....
    go func() {
		for {
			select {
			case <-stop:
				return
			
			....
             }
	}()
    ......
}
```

In gorutine, the external `stop` parameter is not visible to it, so the program is blocked inside this gorutine

Reply:
Please remove the `perf/pulsar-perf-go` bin file